genericshomework/src/main/java/BinaryTree.java

```diff
@@ -0,0 +1,113 @@
+import java.util.List;
+import java.util.Stack;
+
+/**
+ * BinaryTree implementation.
+ */
+
+class BinaryTree<T extends Comparable<T>>{
+    private TreeNode root;
+
+    TreeNode getRoot() {
+        return this.root;
+    }
+
+    // Iterative solution using Stack
+    void inOrderIterative(TreeNode<Integer> root) {
```

Comment:
The method should have JavaDoc that states that it does not return the order, it prints it out to standard output.